### PR TITLE
feat(listing): move access info from menu to byline

### PR DESCRIPTION
**Pull request by hypeJunction**
Entity access information is now rendered in the byline, and not in the
entity menu

listing

### CODE REVIEW

File: views/default/object/elements/access.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * Displays information about the author, the time and the access of the post
```
Comment (hypeJunction) [inline views/default/object/elements/access.php]: Access only

File: views/default/output/access.php [hunk 2]
```diff
@@ -6,48 +6,55 @@
...
 
+switch ($access_id) {
+	case ACCESS_FRIENDS :
+		$class[] = 'elgg-access-friends';
+		break;
+	case ACCESS_PUBLIC :
+		$class[] = 'elgg-access-loggedin';
+		break;
+	case ACCESS_LOGGED_IN :
+		$class[] = 'elgg-access-public';
```
Comment (hypeJunction) [inline views/default/output/access.php]: Reverse

**Comment by mrclay** [comment]
👏

**Comment by jdalsem** [comment]
👍

**Comment by hypeJunction** [comment]
This is ready to go in I think

**Pull request merged**